components/docusign/docusign.app.js

```diff
@@ -0,0 +1,129 @@
+const axios = require("axios");
+
+module.exports = {
+  type: "app",
+  app: "docusign",
+  propDefinitions: {
+    account: {
+      type: "string",
+      label: "Account",
+      description: "",
+      async options() {
+        const { accounts } = await this.getUserInfo();
+        return accounts.map((account) => {
+          return {
+            label: account.account_name,
+            value: account.account_id,
+          };
+        });
+      },
+    },
+    template: {
```

Comment:
Based on Zapier's action, I believe only account and template should be required. Everything else can inherit from the template settings, so I think all other props should default to `optional: true`